synapse/lib/webapp.py

```diff
@@ -102,14 +102,14 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config={}, srv_config={}):
```

Comment:
I am OK with changing the fn signature here to something else, but these configs cannot be combined.